### PR TITLE
Adds Fork me on Github ribbon

### PR DESCRIPTION
**Pull request by sudoankit**
Added a github ribbon in the website using CSS.

Fixes issue #48

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,8 +16,7 @@
   {% if site.touch_icon %}<link rel="apple-touch-icon" href="{{ site.touch_icon }}">{% endif %}
 </head>
 <body>
-  <div id="main">
-
+<a class="github-fork-ribbon" href="https://github.com/coala/coala" title="Fork me on GitHub">Fork me on GitHub</a>
```
Comment (sils) [inline index.html:20]: you removed the div id=main, is that a good idea?

Comment (sudoankit) [reply index.html:20]: Oh well. Sorry. Jekyll build it correctly so I didn't go through it. Let me put it back. Should I rebase or just commit again?

**Comment by sils** [comment]
at this repo we don't care :P

Sincerely,

Lasse Schuirmann

lasse@schuirmann.net
http://coala.io/ - http://viperdev.io/ - http://gitmate.com/

**Comment by sudoankit** [comment]
Thank you. It's done now.

**Pull request merged**